content/manuals/desktop/features/gordon.md

```diff
@@ -0,0 +1,289 @@
+---
+
+{{< introduced desktop 4.37 >}}
+
+{{% restricted title=Beta %}}
+Ask Gordon is a Beta feature. Features, user interface, and behavior are
+subject to change in future releases.
+{{% /restricted %}}
+
+Ask Gordon is your personal AI assistant embedded in Docker Desktop and the
+Docker CLI. It's designed to streamline your workflow and help you make the
+most of the Docker ecosystem.
+
+## What is Ask Gordon?
+
+Ask Gordon is a suite of AI-powered capabilities integrated into Docker's
+tools. These features, currently in Beta, are not enabled by default, and are
+not production-ready. You may also encounter the term "Docker AI" as a broader
+reference to this technology.
+
+The goal of Ask Gordon is to make Docker's tools for managing images and
+containers more intuitive and accessible. It provides contextual assistance
+tailored to your local environment, including Dockerfiles, containers, and
+applications.
+
+Ask Gordon integrates directly with Docker's tools to help you perform specific
+tasks. It understands your local setup, such as your local source code and
+images. For example, you can ask Gordon to help you identify vulnerabilities in
+your project or how to optimize a Dockerfile in your local repository. This
+tight integration ensures responses are practical and actionable.
+
+> [!NOTE]
+> Ask Gordon is powered by Large Language Models (LLMs). Like all LLM-based
+> tools, its responses may sometimes be inaccurate. Always verify the
+> information provided.
+
+### What data does Gordon access?
+
+When you use Ask Gordon, the data it accesses depends on the context of your
+query:
+
+- Local files: If you use the `docker ai` command, Ask Gordon can access
+  files and directories in the current working directory where the command is
+  executed. In Docker Desktop, if you ask about a specific file or directory in
+  the **Ask Gordon** view, you'll be prompted to select the relevant context.
+- Local images: Gordon integrates with Docker Desktop and can view all images
+  in your local image store. This includes images you've built or pulled from a
+  registry.
+
+To provide accurate responses, Ask Gordon may send relevant files, directories,
+or image metadata to the Gordon backend along with your query. This data
+transfer occurs over the network but is never stored persistently or shared
+with third parties. It is used exclusively to process your request and
+formulate a response.
+
+All data transferred is encrypted in transit.
+
+### How your data is collected and used
+
+Docker collects anonymized data from your interactions with Ask Gordon to
+enhance the service. This includes the following:
+
+- Your queries: Questions you ask Gordon.
+- Responses: Answers provided by Gordon.
+- Feedback: Thumbs-up and thumbs-down ratings.
+
+To ensure privacy and security:
+
+- Data is anonymized and cannot be traced back to you or your account.
+- Human reviewers may analyze anonymized data to refine responses for similar
```

Comment:
Please remove that line as seen with Legal.
`Docker collects anonymized data from your interactions with Ask Gordon to
 enhance the service` is enough